continuous_integration/check_wheels.py

```diff
@@ -0,0 +1,33 @@
+"""Print available wheels for a particular Python package."""
+import re
+import sys
+
+import requests
```

Comment:
Where does `requests` come from?

Reply:
It's an indirect dependency of gensim (via `smart_open`).

We don't explicitly list it anywhere like setup.py because this is a utility script.